modules/special-rules/pages/super-heavy-weapons.adoc

```diff
@@ -28,27 +24,34 @@ When you shoot with a detachment that includes super-heavy weapons, you may get
 * For any regular Firepower that your detachment brings to bear on the target detachment, place Blast markers as normal.
 * For the super-heavy weapons that your detachment brings to bear on the target detachment, place Blast markers as follows.
 +
+.Blast marker table
+[cols="^,^2"]
 |===
-^|Number of super-heavy weapons brought to bear ^|Number of Blast markers to place
+^|Number of super-heavy weapons ^| Blast markers placed
```

Comment:
Just a note: You don't need to specify `^` for centre alignment in every cell. It's already set for the columns with the preceding `[cols="^,^2"]`. I'll tidy it up when I review the topic in detail. I'll probably also combine the two related tables into a partial in much the same way as I did for the terrain table, so we can also reuse them for the quick reference.